tests/test_romm.py

```
@@ -0,0 +1,1603 @@
+    assert "Could not reach RomM" in status["error"]
+    # SSOT: the frontend renders its warning from this, not its own copy.
+    assert status["dat_safe_output_exts"] == sorted(DAT_SAFE_OUTPUT_EXTS)
+
+
+# ----------------------------------------------------------------------
+# re-pin queue
+# ----------------------------------------------------------------------
+
+
+@pytest.fixture(name="repin_db")
+def _repin_db(tmp_path: Path):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_repin_db`

(ANN202)